bool new_mission_request
